### DISCUSSION TITLE
Wikipedia

### DISCUSSION THREAD

**Discussion by rodajc** [arendst/Tasmota#20194] [General]
Why is there no article about Tasmota on [Wikipedia](https://www.wikipedia.org/)?

**Comment by barbudor** [comment]
But there's a LinkedIn group 🤣

Feel free to start the page

**Reply by rodajc** [reply]
I would love to, but I am a Tasmota pre-noob, so maybe not such a good idea. Unless you are happy with a page full of errors.
